lib/active_model/serializer/concerns/configuration.rb

```diff
@@ -29,6 +29,7 @@ def config.array_serializer
         # Make JSON API top-level jsonapi member opt-in
         # ref: http://jsonapi.org/format/#document-top-level
         config.jsonapi_include_toplevel_object = false
+        config.include_data_default = true
```

Comment:
current default is true. per the PR description, I vote we change that default but up to you guys